eng/scripts/CodeChecks.ps1

```diff
@@ -62,6 +62,22 @@ try {
                     }
         }
 
+    Write-Host "Validating sample readmes"
```

Comment:
@KarishmaGhiya is soon going to be integration some common scripts into the repo that will validate links and as well as ensure they are not relative. So if this can way I'd suggest we let here work handle this.

Reply:
For reference the script she is going to use to validate links is at https://github.com/Azure/azure-sdk/blob/master/eng/scripts/Verify-Links.ps1, but we plan to move it into azure-sdk-tools and thus under eng/common in this repo at that point.

Reply:
Yeah, the validation piece can wait. I will leave this PR open in case I get a chance to do an update of all of the offending sample readmes. I won't merge in the change to CodeChecks though.

Reply:
On unrelated note, @weshaggard why can't we automate the link conversion process instead (pre process md files before publishing and avoid making everyone think about it)?

Reply:
@scbedd and I had been talking about the automation part, and he was also speaking with the MS Docs team that owns the extraction tool.  When last I heard, we were in a holding pattern because we needed an extension to their extraction process to inject behavior unless we wanted to write our own pre-processor.   Scott may have fresher information.

Reply:
I we chose between a validation tool and preprocessor I would pick preprocessor and make developers lives easier.

Reply:
Apologies; what I was describing was having a translation process that ran as part of the Docs standard tooling or wrapping the tooling with our own preprocessor.   I wasn't referring to a validator.

Reply:
Yes we have done a bunch of thinking and re-thinking on this subject and I've called out a number of issues in https://github.com/Azure/azure-sdk/issues/1214. The general guidance is to use links to master and we for the consumption side we control we will re-write the links to use the release tags. So we will have a mix of both and we will have some validation that will catch the relative links to try and ensure we stay clean. If needed we can also add some helpers to convert the relative links to master links but I believe our best option is to keep full absolute links to master as our resting state in the repo.